javascripts/discourse/components/modal/dev-toolbox.gjs

```diff
@@ -0,0 +1,178 @@
+import DModal from "discourse/components/d-modal";
+import DiscourseURL from "discourse/lib/url";
+import dIcon from "discourse-common/helpers/d-icon";
+import i18n from "discourse-common/helpers/i18n";
+import ComboBox from "select-kit/components/combo-box";
+
+const COMMON_SETTINGS = [
+  { id: "branding", name: "branding", path: "branding" },
+  {
+    id: "categoryStyle",
+    name: "category_style",
+    path: "basic?filter=category_style",
+  },
+  { id: "tagStyle", name: "tag_style", path: "all_results?filter=tag_style" },
+  { id: "topMenu", name: "top_menu", path: "all_results?filter=top_menu" },
+  {
+    id: "desktopCategoryPageStyle",
+    name: "desktop_category_page_style",
+    path: "basic?filter=desktop_category_page_style",
+  },
+  {
+    id: "navigationMenu",
+    name: "navigation_menu",
+    path: "all_results?filter=navigation_menu",
+  },
+];
+
+export default class DevToolboxModal extends Component {
+  @tracked commonSetting;
+
+  @action
+  toggleAlerts() {
+    const hidden = document.body.classList.contains("alerts-hidden");
+
+    localStorage.setItem(
+      "alerts-visibility",
+      hidden ? "alerts-visible" : "alerts-hidden"
+    );
+    document.body.classList.toggle("alerts-hidden");
+    this._triggerModalClose();
+  }
+
+  @action
+  togglePluginOutlets() {
+    const visible = document.body.classList.contains("plugin-outlets-visible");
+
+    localStorage.setItem(
+      "plugin-outlet-visibility",
+      visible ? "outlets-invisible" : "outlets-visible"
+    );
+    document.body.classList.toggle("plugin-outlets-visible");
+    this._triggerModalClose();
+  }
+
+  @action
+  navigateToSetting(setting) {
+    const base = "/admin/site_settings/category";
+    const { path } = COMMON_SETTINGS.find((s) => s.id === setting);
+    DiscourseURL.routeTo(`${base}/${path}`);
+    this.commonSetting = setting;
+  }
+
+  _triggerModalClose() {
+    if (settings.actions_close_modal) {
+      this.args.closeModal();
+    }
+  }
+
+  <template>
+    <DModal
+      @closeModal={{@closeModal}}
+      @title={{i18n (themePrefix "dev_utils.modal.title")}}
+      class="dev-toolbox-modal"
+    >
+      <h3>{{dIcon "running"}}
+        {{i18n (themePrefix "dev_utils.actions.title")}}</h3>
+      <div class="modal-button-group">
+        <DButton
+          @title={{themePrefix "dev_utils.actions.toggle_alerts"}}
+          @icon="exclamation"
+          @label={{themePrefix "dev_utils.actions.toggle_alerts"}}
+          @action={{this.toggleAlerts}}
+        />
+        <DButton
+          @title={{themePrefix "dev_utils.actions.toggle_plugin_outlets"}}
+          @icon="plug"
+          @label={{themePrefix "dev_utils.actions.toggle_plugin_outlets"}}
+          @action={{this.togglePluginOutlets}}
+        />
+      </div>
+
+      <h3>{{dIcon "link"}} {{i18n (themePrefix "dev_utils.links.title")}}</h3>
+      <div class="modal-button-group">
+        <DButton
+          @title={{themePrefix "dev_utils.links.themes"}}
```

Comment:
One minor thing - if titles are the same as labels then there's probably no need for them?